util/hot_backup.h

```diff
@@ -0,0 +1,127 @@
+// -------------------------------------------------------------------
+//
+// hot_backup.h
+//
+// Copyright (c) 2011-2016 Basho Technologies, Inc. All Rights Reserved.
+//
+// This file is provided to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file
+// except in compliance with the License.  You may obtain
+// a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// -------------------------------------------------------------------
+
+#ifndef STORAGE_LEVELDB_INCLUDE_HOT_BACKUP_H_
+#define STORAGE_LEVELDB_INCLUDE_HOT_BACKUP_H_
+
+#include "util/thread_tasks.h"
+
+namespace leveldb
+{
+
+namespace config {
+
+// how many hot backup directories:  backup, backup.1, ... backup.(kNumBackups-1)
+static const int kNumBackups = 5;
```

Comment:
Going to wait on this.  The "config" namespace is established in db/dbformat.h for several values that could also become configurable some day ... but have not.  Going to leave this static until there is a customer requirement.